docs/guides/akamai/get-started/observability-with-datastream-and-trafficpeak/index.md

```diff
@@ -0,0 +1,93 @@
+
+This guide looks at the TrafficPeak observability solution and reviews a tested, proven observability architecture built for a high-traffic delivery platform. This solution combines Akamai’s edge-based DataStream log streaming service, SIEM integration, and TrafficPeak built on Linode cloud infrastructure to support large-scale traffic, logging, and data retention.
+
+## TrafficPeak On Akamai Cloud
+
+### What Is TrafficPeak?
+
+TrafficPeak is a fully managed observability solution that works with DataStream log streaming and Akamai Cloud Computing. TrafficPeak is managed and hosted by Hydrolix, and uses Linode Compute Instances alongside Linode Object Storage for data processing and storage. With TrafficPeak, customers are provided with access to a Grafana interface with preconfigured, customizable dashboards for data visualization and monitoring.
+
+### Who Is TrafficPeak For?
+
+TrafficPeak is for Akamai customers that need an all-in-one, cost-effective, turnkey observability solution for large, petabyte-scale volumes of data.
+
+For more detailed information on features and pricing, see: [Observability on Akamai cloud computing: TrafficPeak](https://hydrolix.io/partner-program/trafficpeak/)
+
+## Overcoming Challenges
+
+### Cost Reduction & Visibility
+
+*Reduce observability costs by up to 75% while standardizing data visualization.*
+
+Where other observability solutions are built for application-specific observability, TrafficPeak is built to achieve CDN-level observability supporting extremely large amounts of data. TrafficPeak is designed to work with DataStream to ingest logs from the edge and uses Compute Instances with Object Storage to achieve an efficient, decoupled, and stateless architecture. By keeping traffic and CDN logs on the same platform, TrafficPeak can reduce your observability costs by up to 75%.
+
+With large amounts of data, the need for standardizing data reporting becomes even more important. TrafficPeak’s Grafana dashboard supports as many users you may need, providing a centralized data source. This helps streamline organizational operations and allows developers that work with the data the ability to all get it from the same place.
+
+### Large Data Volumes & Data Retention
+
+*Address data volume and retention challenges with an observability solution on the same platform as your edge delivery.*
+
+There are numerous cost and infrastructure-based challenges that are associated with large volumes of data, including: data acquisition, data storage and retention, data searching and efficiency, and more. Since TrafficPeak is built for Akamai, you can address these challenges by coupling your edge and observability solutions on a single platform.
```

Comment:
On line 41 we describe that the architecture is 'decoupled'. On this line we describe an aspect of it that is 'coupled'. I think it would be worth using a different word than 'coupled' to avoid ambiguity

It might also be a good idea to give a brief description or link to a description of what 'decoupled' and 'stateless' mean in this context on line 41

Reply:
changed to "combining"